### ISSUE TITLE
Media strategy outputting malformed css

### ISSUE THREAD

**Issue by esadewater** [JavierM42/tailwind-mode-aware-colors#2]
Hi there,

I found a problem with Tailwinds media dark mode strategy. Whenever I try to use `darkMode: 'media'` together with the `tailwind-mode-aware-colors` plugin, it outputs a malformed css for the dark mode colors. It isn't wrapping the color vars in the `html { }` selector. This only happens for the media dark mode strategy, while the class strategy is working. I'm using Vite as asset bundler.

My `tailwind.config.js` looks like this:
```js
const colors = require('./resources/tailwind/colors');

/** @type {import('tailwindcss').Config} */
module.exports = require('tailwind-mode-aware-colors')({
    theme: {
         colors: {
            'primary': {
                light: colors.primary['40'],
                dark: colors.primary['80'],
            },
            'on-primary': {
                light: colors.primary['100'],
                dark: colors.primary['20']
            }
            ...
        }
    }
});
```

The colors in the array:
```js
module.exports = {
    primary: {
        100: '#ffffff',
        99: '#fffbff',
        98: '#fff8f3',
        95: '#ffeeda',
        ...
    },
    ...
}
```

And the generated css looks like this:
```css
html {
    --color-primary: 127 86 0;
    --color-on-primary: 255 255 255;
    ...
}

@media (prefers-color-scheme: dark) {
    --color-primary: 252 187 74;
    --color-on-primary: 67 44 0;
    ...
    {}
}
```

While the expected css would look like this:
```css
html {
    --color-primary: 127 86 0;
    --color-on-primary: 255 255 255;
    ...
}

@media (prefers-color-scheme: dark) {
    html {
        --color-primary: 252 187 74;
        --color-on-primary: 67 44 0;
        ...
    }
}
```

**Comment by JavierM42** [comment]
Good catch! Oops, I never noticed that because I always use `class` and JS-based toggles.

I've released v1.2.3 which should fix the issue.

**Comment by esadewater** [comment]
Awesome, that perfectly works for me. Thank you :)
